### ISSUE TITLE
Document Accessibility Improvement

### ISSUE THREAD

**Issue by ghost** [discordextremelist/website#213]
This page that appears after clicking the "Login With Discord" button has some text that says "By clicking continue you agree to our Terms of Service, Privacy Policy, and Website Guidelines.". As an accessibility improvement, those documents (the ToS, Privacy Policy, & Website Guidelines) should be linked in that text via hyperlinks to make it easier for people to go to those documents.
